src/apps/loyalty/server/index.tsx

```diff
@@ -82,7 +82,7 @@ app.get("/inquiries", (req, res) => {
         styles,
         html,
         entrypoint: "/bundles/inquiries.js",
-        bootstrapData: `var DATA = ${JSON.stringify(data)};`,
+        bootstrapData: `var DATA = ${JSON.stringify(data)}; var USER_DATA = ${JSON.stringify(req.user.toJSON())};`,
```

Comment:
Ah, I remember now, we need to harden this against XSS attacks. In the past I came across [this lib of yahoo](https://github.com/yahoo/serialize-javascript) that should [be able to escape HTML](https://github.com/yahoo/serialize-javascript#automatic-escaping-of-html-characters).